libsplinter/src/orchestrator/mod.rs

```diff
@@ -332,6 +332,41 @@ impl ServiceOrchestrator {
             .collect())
     }
 
+    /// Create a service that has previously been stopped according to the specified definition.
+    /// The arguments provided must match those required to create the service.
+    pub fn add_stopped_service(
```

Comment:
Should this just be an option to add service?

Reply:
Well, the other `initialize_service` option in the orchestrator that adds a service would need another argument to tell that the service is 'stopped' because the orchestrator isn't really aware about that status, and is easier to handle from the admin service's perspective (hence the separate function)